com.unity.cinemachine/Runtime/Core/TargetPositionCache.cs

```diff
@@ -211,11 +219,15 @@ static void CreatePlaybackCurves()
         const float kWraparoundSlush = 0.1f;
 
         /// <summary>
-        /// If Recording, will log the target position at the CurrentTime.
-        /// Otherwise, will fetch the cached position at CurrentTime.
+        /// When using Timeline in Edit mode:
+        ///  - If Recording, will log the target position at the CurrentTime.
+        ///  - Otherwise, will fetch the cached position at CurrentTime.
+        ///  
+        /// In Play mode, and when not scrubbing timeline:
+        ///  - will return the position directly from the transform.
```

Comment:
```suggestion
        /// When using Timeline in Edit mode:
        ///  - If you're Recording, the method logs the target position at the CurrentTime.
        ///  - Otherwise, it fetches the cached position at CurrentTime.
        ///  
        /// When using Timeline in Play mode, and when you're not scrubbing it:
        ///  - The method returns the position directly from the Transform.
```